server/embed/etcd.go

```diff
@@ -719,6 +719,7 @@ func (e *Etcd) serveClients() (err error) {
 			Time:    e.cfg.GRPCKeepAliveInterval,
 			Timeout: e.cfg.GRPCKeepAliveTimeout,
 		}))
+		gopts = append(gopts, e.cfg.GRPCAdditionalServerOptions...)
```

Comment:
Does it need to move out this if statement?

Reply:
+1